src/legends/searchable-discrete-color-legend.js

```diff
@@ -59,24 +60,26 @@ function SearchableDiscreteColorLegend(props) {
     null;
   const filteredItems = searchFn(items, searchText);
   return (
-    <div className={`rv-search-wrapper ${className}`} style={{width, height}}>
-      <form className="rv-search-wrapper__form">
-        <input
-          type="search"
-          placeholder={searchPlaceholder}
-          className="rv-search-wrapper__form__input"
-          value={searchText}
-          onChange={onChange}/>
-      </form>
-      <div className="rv-search-wrapper__contents">
-        <DiscreteColorLegend
-          colors={colors}
-          items={filteredItems}
-          onItemClick={onItemClick}
-          orientation={orientation}
-        />
+    <LegendContainer className={className}>
+      <div className="rv-search-wrapper" style={{width, height}}>
```

Comment:
Hi @mcnuttandrew, 

Just a heads up, I didn't move `rv-search-wrapper` in this `div` to `<LegendContainer>` due to a known weakness of styled-components as we have discussed in #823 - conflict of specificity. 

If we do it, the css specified on class `rv-search-wrapper` will be broken. Anyway, we have set the custom class name at the top tag level. I'd rather use `LegendContainer` as a wrapper on top of everything. The custom class name is set at the top level so users can modify any css within this component.

Details: https://www.styled-components.com/docs/advanced#existing-css